ntripclient.c

```diff
@@ -71,7 +71,7 @@
 #define MAXDATASIZE 1000 /* max number of bytes we can get at once */
 
 /* CVS revision and version */
-static char revisionstr[] = "$Revision: 1.6.1 $";
+static char revisionstr[] = "$Revision: 1.6.2 $";
```

Comment:
We can keep the version as 1.6.1 since we haven't made a 1.6.1 release tag yet

Reply:
done